src/app/core/admin/admin-overview/admin-overview.component.ts

```diff
@@ -51,7 +51,8 @@ export class AdminOverviewComponent implements OnInit {
    * Send a reference of the PouchDB to the browser's developer console for real-time debugging.
    */
   debugDatabase() {
-    console.log(this.db);
+    console.log(this.dbResolver);
+    console.log(this.dbResolver.getDatabase());
```

Comment:
Can be removed in production?

Reply:
This exceptional console log is actually intentional: Make the Database service available in the console for scripting. I added better explanation for this here.